scripts/generate_blobs.py

```diff
@@ -21,149 +21,50 @@
 loaded into the target MCU RAM. Generates files compatible with C programs 
 and python programs (DAPLink Interface Firmware and pyDAPFlash)
 '''
-from struct import unpack
-from os.path import join
-from jinja2 import Template, StrictUndefined
-import sys, os, collections
+import os
+import argparse
+from flash_algo import PackFlashAlgo
 
 # TODO
 # FIXED LENGTH - remove and these (shrink offset to 4 for bkpt only)
-# defines better documented
-ALGO_OFFSET = 0x20
-BLOB_START = 0x20000000
 BLOB_HEADER = '0xE00ABE00, 0x062D780D, 0x24084068, 0xD3000040, 0x1E644058, 0x1C49D1FA, 0x2A001E52, 0x4770D1F2,'
```

Comment:
We should see if we can get simplify this blob header. We should only need the first word. The rest of the code is something like a CRC algorithm.